packages/sitecore-jss/src/utils/app-root-resolver.ts

```diff
@@ -0,0 +1,52 @@
+import { GraphQLRequestClient } from '../graphql-request-client';
```

Comment:
Still not sure where this thing fits :) This one's pretty different than the other "utils" under this folder, and it's not obvious by the exported `getAppRoot` name that it's gql-specific (requires a `GraphQLRequestClient` parameter even). What do you think about a something like a root `graphql-utils` and putting this here?

Reply:
I'm fine with this here or `graphql-utils`, maybe re-visit with next PR.